### ISSUE TITLE
xml to cryxmlb

### ISSUE THREAD

**Issue by systemsiteseason** [pleonex/HappyXml#2]
do you compile xml to cryxml? some game cant load xml (homefront the revolution)

**Comment by pleonex** [comment]
No, this program only can export data (from CryXML to XML). It cannot import back.
